src/ontology/mondo-ingest-odk.yaml

```diff
@@ -41,13 +41,13 @@ components:
       source: http://www.orphadata.org/data/ORDO/ordo_orphanet.owl
 robot_report:
   release_reports: False
-  fail_on : None
```

Comment:
## Remove space
For internal consistency, I removed the space in front of some of the `:` in this file.

I double checked on Google (no quick results) and [w/ ChatGPT](https://chat.openai.com/share/ed90a666-35ed-448e-9d32-7ef23118a675) as to whether this was syntactically the same and that this wouldn't break anything.